packs/data/bestiary-effects.db/effect-aura-of-protection-solar.json

```
@@ -0,0 +1,74 @@
+{
+    "_id": "s16XQpDz2HNzR9BB",
+    "img": "systems/pf2e/icons/spells/dancing-shield.webp",
+    "name": "Effect: Aura of Protection (Solar)",
+    "system": {
+        "badge": null,
+        "description": {
+            "value": "<p>Allies in the solar's aura gain a +2 status bonus to AC against evil creatures and a +2 status bonus to saves against effects from evil creatures. The bonus increases to +4 against control by evil creatures and attacks by evil summoned creatures.</p>"
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "unlimited",
+            "value": -1
+        },
+        "level": {
+            "value": 1
+        },
+        "rules": [
+            {
+                "key": "FlatModifier",
+                "predicate": [
+                    "origin:trait:evil"
+                ],
+                "selector": [
+                    "ac",
+                    "saving-throw"
+                ],
+                "type": "status",
+                "value": 2
+            },
+            {
+                "key": "FlatModifier",
+                "predicate": [
+                    "origin:trait:evil",
+                    {
+                        "or": [
+                            "controlled",
```

```suggestion
                            "inflicts:controlled",
```
since controlled isn't a trait we use this for now. One day we will find a way to automagically handle this.